packages/mobile/src/invite/saga.ts

```diff
@@ -82,18 +83,36 @@ export function getInvitationVerificationFeeInWei() {
 }
 
 export async function generateLink(inviteCode: string, recipientName: string) {
-  const packageName = VersionCheck.getPackageName().replace(/\.debug$/g, '.integration')
-  const playStoreLink = await VersionCheck.getPlayStoreUrl({ packageName })
+  const AndroidPackageName = VersionCheck.getPackageName().replace(/\.debug$/g, '.integration')
```

Comment:
why are we defaulting to `integration` here?

Reply:
I guess the reason is that the build we use for internal testing is `integration`, but not sure.